contrib/subrepo-cheri-libcxxrt/src/dwarf_eh.h

```diff
@@ -437,6 +437,10 @@ static bool dwarf_eh_find_callsite(struct _Unwind_Context *context,
 	result->landing_pad = 0;
 	// The current instruction pointer offset within the region
 	uint64_t ip = (char*)_Unwind_GetIP(context) - (char*)_Unwind_GetRegionStart(context);
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(__aarch64__)
+	// Account for the LSB in the address returned by _Unwind_GetIP().
+	ip--;;
```

Comment:
I'd probably prefer masking via:
```suggestion
	ip &= ~(uint64_t)1;
```
(but if not, you have an extra semicolon)

Reply:
So `ip` here is the delta, not the raw PC, so masking isn't quite right though it might work in practice.  I thought about having a larger change where I did mask off the value of _Unwind_GetIP(), but that gave a larger diff.

Reply:
Hm, shouldn't _Unwind_GetRegionStart include the LSB then...

Reply:
No the region start is something like "all of text".  It definitely had LSB clear when I was single stepping.